lib/active_model/serializer.rb

```diff
@@ -37,6 +37,7 @@ class << self
       attr_accessor :_cache_except
       attr_accessor :_cache_options
       attr_accessor :_cache_digest
+      attr_accessor :_type
```

Comment:
- [ ] plz remove this accessor as the class_attribute below duplicates the behavior